dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -361,33 +387,14 @@ fi
 echo
 echo "[CHE] Deploying Che on ${OPENSHIFT_FLAVOR} (image ${CHE_IMAGE})"
 
-
 DEFAULT_CHE_DEPLOYMENT_FILE_PATH=./che-openshift.yml
 CHE_DEPLOYMENT_FILE_PATH=${CHE_DEPLOYMENT_FILE_PATH:-${DEFAULT_CHE_DEPLOYMENT_FILE_PATH}}
+DEFAULT_CHE_CONFIG_MAP_FILE_PATH="$(pwd)/che-config-map"
```

Comment:
Can you check whether this fails in case user call deploy script, not from the current directory?

Reply:
done